oximeter/oximeter/schema/bfd-session.toml

```diff
@@ -0,0 +1,101 @@
+format_version = 1
+
+[target]
+name = "bfd_session"
+description = "A Bidirectional Forwarding Protocol (BFD) session"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "hostname", "rack_id", "sled_id", "peer" ] },
+]
+
+[[metrics]]
+name = "control_packet_send_failures"
+description = "Total number of failures to send a control packets to a peer"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ ] }
+]
+
+[[metrics]]
+name = "control_packets_sent"
+description = "Total number of control packets sent to a peer"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ ] }
+]
+
+[[metrics]]
+name = "control_packets_received"
+description = "Total number of control packets received from a peer"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ ] }
+]
+
+[[metrics]]
+name = "message_receive_error"
```

Comment:
In the existing definition, this is misspelled as `message_recieve_error`. I've renamed it here, which means it's really a new timeseries. I'll start using the new one in the BFD implementation, and then I'll make a pass to remove defunct timeseries which would include the original, misspelled one.